src/main/java/com/dnd/runus/application/badge/BadgeService.java

```diff
@@ -52,36 +57,46 @@ public AchievedBadgesResponse getAchievedBadges(long memberId) {
     @Transactional(readOnly = true)
     public AllBadgesListResponse getListOfAllBadges(long memberId) {
 
-        List<BadgeWithAchieveStatusAndAchievedAt> allBadges =
-                badgeRepository.findAllBadgesWithAchieveStatusByMemberId(memberId);
+        List<BadgeAchievement.OnlyBadge> allBadges =
+                badgeAchievementRepository.findByMemberIdOrderByBadgeTypeAndAchievedAt(memberId);
 
+        // 최신 획득한 뱃지(오늘 기준으로 일주일)
         LocalDateTime oneWeekAgo = LocalDate.now(SERVER_TIMEZONE_ID)
                 .atStartOfDay(SERVER_TIMEZONE_ID)
                 .toOffsetDateTime()
                 .minusDays(7)
                 .toLocalDateTime();
 
-        EnumMap<BadgeType, List<BadgeWithAchievedStatus>> badgeMap = allBadges.stream()
+        List<AchievedBadge> recencyBadges = allBadges.stream()
+                .filter(v -> oneWeekAgo.isBefore(v.createdAt().toLocalDateTime()))
+                .map(v -> new AchievedBadge(
+                        v.badge().badgeId(),
+                        v.badge().name(),
+                        v.badge().imageUrl(),
+                        v.createdAt().toLocalDateTime()))
+                .toList();
+
+        // badgeType별 획득한 배지
+        EnumMap<BadgeType, List<AchievedBadge>> badgesWithType = allBadges.stream()
```

Comment:
Collection 자료형을 저장할 땐 interface 타입을 사용하는게 객체지향 관점에서 좋을 것 같아요!
- https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java

```suggestion
        Map<BadgeType, List<AchievedBadge>> badgesWithType = allBadges.stream()
```